src/starspace.cpp

```diff
@@ -299,19 +300,44 @@ void StarSpace::loadBaseDocs() {
 void StarSpace::predictOne(
     const vector<Base>& input,
     vector<Predictions>& pred) {
+      predict(input, pred, args_->K);
+}
+
+void StarSpace::predict(
```

Comment:
Could you add some test around this function?

Reply:
Whoops, unfortunately things goes wrong and i closed this PR after merging changes. So, the new one is here https://github.com/facebookresearch/StarSpace/pull/279

Tests are coming soon